### ISSUE TITLE
Failover client logs an error

### ISSUE THREAD

**Issue by dim** [go-redis/redis#500]
On client init, I can see the following error:

```
sentinel: ReceiveMessage failed: ERR wrong number of arguments for 'ping' command
```

I tried to debug this but not sure how this would happen. Any idea?

**Comment by vmihailenco** [comment]
Should be fixed by https://github.com/go-redis/redis/pull/486. It happens when you start receiving notifications without subscribing to any channels. This way `PING` command returns slightly different response which was not handled correctly.

**Comment by dim** [comment]
Still happens on latest master, only on FailoverClient

**Comment by vmihailenco** [comment]
Probably just an old Redis Server that does not support `PING hello`?

**Comment by dim** [comment]
Hmm, that's probably right, it's v3.0.x? Do we need the extra message on
ping?

On 23 Feb 2017 1:11 pm, "Vladimir Mihailenco" <notifications@github.com>
wrote:

Probably just an old Redis Server that does not support PING hello?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/go-redis/redis/issues/500#issuecomment-281987675>, or mute
the thread

**Comment by vmihailenco** [comment]
Not really - should be fixed on master and v5.2.6.
